doc/release/upcoming_changes/25233.new_feature.rst

```diff
@@ -0,0 +1,10 @@
+``ndarray.device`` and ``ndarray.to_device``
+--------------------------------------------
+
+``ndarray.device`` attribute and ``ndarray.to_device`` function were
+added to `numpy.ndarray` class for Array API compatibility.
+
+Additionally, ``device`` keyword-only arguments were added to:
+`numpy.asarray`, `numpy.arange`, `numpy.empty`, `numpy.empty_like`,
+`numpy.eye`, `numpy.full`, `numpy.full_like`, `numpy.linspace`,
+`numpy.ones`, `numpy.ones_like`, `numpy.zeros`, and `numpy.zeros_like`.
```

Comment:
Sorry for not noticing this sooner, can we say in this release note that only `device=cpu` is supported?